README.md

```diff
@@ -22,11 +22,12 @@ df = DataFrame(species=["E. coli", "B. fragilis", "L. casei"],
                sample2=[0.3, 0.7, 0.0],
                sample3=[0.0, 0.2, 0.8])
 
-abund = AbundanceTable(df)
+abund = abundancetable(df)
```

Comment:
I've made an `abundancetable` function that creates a `ComMatrix` but has exactly the same syntax as the current `AbundanceTable` constructors - for easy conversion.